src/System.Private.ServiceModel/tools/test/Bridge/Bridge/ConfigController.cs

```diff
@@ -16,14 +16,25 @@ namespace Bridge
 {
     public class ConfigController : ApiController
     {
-        private const int Default_BridgeIdleTimeoutMinutes = 20;
+        public static EventHandler<Tuple<BridgeConfiguration, BridgeConfiguration>> BridgeConfigurationChanged;
```

Comment:
I would recommend just defining this. I personally don't like Tuples since they Item1 and Item2 is just :scream: 
Its your call and don't block on this comment. 

``` 
struct ChangedEventArgs<T> : EventArgs
{ 
    public T Previous {get; set;}
    public T Updated {get; set;}
}
```